subgraphs/_reference_/src/sdk/protocols/perpfutures/pool.ts

```diff
@@ -281,8 +288,7 @@ export class Pool {
    * @param delta The change in total value locked for this pool.
    */
   addVolume(delta: BigDecimal): void {
-    this.pool.cumulativeVolumeUSD = this.pool.cumulativeVolumeUSD.plus(delta);
-    this.save();
+    this.setVolume(this.pool.cumulativeVolumeUSD.plus(delta));
```

Comment:
To me it seems like setVolume is not really necessary with this change. This line could really just be:

```suggestion
    this.pool.cumulativeVolumeUSD = this.pool.cumulativeVolumeUSD.plus(delta));
    this.save();
```